File: 1413. Minimum Value to Get Positive Step by Step Sum/_1413.py
class Solution:
    def minStartValue(self, nums) -> int:
        res = 0
        tmp = 0
        for i in nums:
            tmp += i
            if tmp<res:
                res = tmp

        return (res-1)*-1

a = Solution()
print(a.minStartValue([1,-2,-3]))

File: 1413. Minimum Value to Get Positive Step by Step Sum/1413.c
int minStartValue(int* nums, int numsSize){
    int res = 0;
    int tmp = 0;

    for(int i=0; i<numsSize; i++){
        tmp += *(nums+i);
        if(tmp<res){
            res = tmp;
        }
    }

    return (res-1)*-1;
}